templates/js-cypress/cypress/e2e/first-spec.cy.js

```diff
@@ -0,0 +1,7 @@
+describe('first tests suite', () => {
+    it('visit Apify main page', () => {
+        cy.visit('/');
+        cy.log('Visiting Apify main page');
+        cy.contains('h1', 'Build reliable web scrapers. Fast.').should('exist');
```

Comment:
Wouldn't be better to use something more generic that won't break if we change the title on the web?

Reply:
sure, changed to 
`cy.contains('div', 'Apify').should('exist');`
I suppose that would be safer. Or any other ideas are welcome :-)